campari/campari_runner.py

```
@@ -186,7 +186,12 @@ def decide_run_mode(self):
 
         # Option 1, user passes a file of SNIDs
         if self.SNID_file is not None:
-            self.SNID = pd.read_csv(self.SNID_file, header=None).values.flatten().tolist()
+            self.SNID = []
+            with open(self.SNID_file, "r") as ifp:
+                for line in ifp:
+                    # Check that the line is non-empty, not a comment, and doesn't contain letters.
+                    if len(line.strip()) > 0 and line[0] != "#" and not any(c.isalpha() for c in line):
```

Do you want to skip letters, or do you want to error out if there are letters?  Right now it will just skip anything with letters, but it might be better to error out, because that means something's wrong with the file.  Plus, if what you need is an integer, there are lots of thing other than letters that are bad.

I would also do: `line.strip()[0] != '#'`, so that lines that start with spaces and then a `#` get skipped.

So, maybe:

```
line = line.strip()
if ( len(line) > 0 ) and ( line[0] != '#' ) and all( c in '0123456789' for c in line ):
```

if what you *really* want is to silently ignore lines with illegal characters, rather than have that be an error.  But, I would recommend making invalid lines an error rather than something to be skipped.  In that case, just let it thrown an exception when you do `int(line)` below, so  you end up with:
```

line = line.strip()
if ( len(line) > 0 ) and ( line[0] != '#' ):
```